src/components/calcite-stepper-item/calcite-stepper-item.scss

```diff
@@ -174,10 +165,10 @@
   border-top-color: rgba($h-bb-060, 0.75);
```

Comment:
See above.

Reply:
Not sure how to replace this with utils

Reply:
We may need to add some additional config to do this.